docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -267,6 +271,46 @@ For IBM Cloud account access management, consult the [IBM Cloud Identity and Acc
 
 For Docker EE cluster access management, use the [UCP Access Control documentation](/datacenter/ucp/2.2/guides/access-control/).
 
+## Secure public network access to swarms
```

Comment:
"Secure" as a verb is super ambiguous especially in headings. It looks like you might mean "Secure public access" as a noun phrase describing "access", but I think you mean "Cause public access to the swarm to be secure". Not sure what the best rewrite is here.